src/bluetooth_linux.c

```diff
@@ -0,0 +1,673 @@
+#include <sys/types.h>
+#include <systemd/sd-bus.h>
+#include <systemd/sd-bus-vtable.h>
+
+#include "fido.h"
+#include "fido/param.h"
+
+#define FIDO_SERVICE_UUID "0000fffd-0000-1000-8000-00805f9b34fb"
+#define FIDO_STATUS_UUID  "f1d0fff2-deaa-ecee-b42f-c9ba7ed623bb"
+#define FIDO_CONTROL_POINT_UUID "f1d0fff1-deaa-ecee-b42f-c9ba7ed623bb"
+#define FIDO_CONTROL_POINT_LENGTH_UUID "f1d0fff3-deaa-ecee-b42f-c9ba7ed623bb"
+#define FIDO_SERVICE_REVISION_UUID "f1d0fff4-deaa-ecee-b42f-c9ba7ed623bb"
+
+#define DBUS_CHAR_IFACE "org.bluez.GattCharacteristic1"
+#define DBUS_DEV_IFACE "org.bluez.Device1"
+#define DBUS_SERVICE_IFACE "org.bluez.GattService1"
+#define DBUS_PROFILE_IFACE "org.bluez.GattProfile1"
+#define DBUS_ADAPTER_IFACE "org.bluez.Adapter1"
+#define DBUS_GATTMANAGER_IFACE "org.bluez.GattManager1"
+
+static sd_bus_slot *profile_slot = NULL;
+static sd_bus_slot *profile_manager_slot = NULL;
```

Comment:
Do we need these or can a floating bus slot be used instead?

Reply:
well, if libfido only looks for connected devices, than this stuff can be completely removed.